chart-widget/template/src/site/widgets/custom-elements/chart/panel.tsx

```diff
@@ -1,14 +1,13 @@
 import React, { useState, useEffect } from 'react';
-import { WixProvider, useWixModules } from '@wix/sdk-react';
-import { editor, widget } from '@wix/editor';
+import { widget } from '@wix/editor';
 import { SidePanel, WixDesignSystemProvider } from '@wix/design-system';
 import '@wix/design-system/styles.global.css';
 import { DEFAULT_TYPE, DEFAULT_ITEMS, type ChartItem } from './common.js';
 import Slice from './panel/slice.js';
 import { ChartType } from './panel/chart-type.js';
 
-function Panel() {
-  const { setProp, getProp } = useWixModules(widget);
+export default () => {
```

Comment:
```suggestion
const Panel: FC = () => {
```
Consider `FC` type usage to be consistent with `cli-app` template

Reply:
Thanks, done